_develop/connect/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Overview
+{:.no_toc}
+
+* TOC
+{:toc}
+
+Blockstack Connect is a Javascript library for integrating your application with Stacks v2. With Connect, you get some big benefits:
+
+- Pre-built UI to introduce your users to Blockstack, before they authenticate
+- Integration with our new authenticator
+- Ability to interact with the Stacks 2.0 blockchain, with support for smart contracts and other transactions.
+
+## Demo
+
+To get a test feel for the user experience of using Connect, you can use [Banter](https://banter.pub).
+
+## How does this compare to `blockstack.js`?
+
+Although [`blockstack.js`](https://github.com/blockstack/blockstack.js) exposes everything you need to handle authentication with Blockstack, there is still the hard problem of getting users used to the paradigm of authentication that is privacy-first and self-sovereign. Many apps implement their own dialogues before authentication, which explain what Blockstack is and why they use it.
+
+`@blockstack/connect` provides developers with a plug-and-play API that is simple to use, and provides great, out-of-the-box education that end-users can understand.
```

Comment:
I don't feel like the question is answered? Why is this connect mechanism not in blockstack.js for a good developer experience? Two competing auth methods shouldn't be there. The mental overhead to implement auth in Blockstack just went up significantly I think

Reply:
Main reason is that Connect includes UI, which bundles React. We don't want to include all of that into blockstack.js. Better might be to include the auth methods in some light library or blockstack.js, and change all docs to _only_ point to that.

Reply:
I'd love to chat about that. I don't yet understand if we're now replacing the auth methods in blockstack.js? If so, we have to deprecate them. More options for the same goals are confusing.

On a higher level, I believe devs liked that there's just one simple lib to use. This will now be different, right? Let's take this discussion offline - outside of this scope :)